Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -385,6 +385,31 @@ void ACesium3DTileset::SetTilesetSource(ETilesetSource InSource) {
   }
 }
 
+bool CompareMaps(
+    const TMap<FString, FString>& Lhs,
+    const TMap<FString, FString>& Rhs) {
+  if (Lhs.Num() != Rhs.Num()) {
+    return false;
+  }
+
+  for (auto& [Key, Value] : Lhs) {
```

Comment:
I think this needs to be `const auto&` or else it will copy the key and value. I think.